zhaquirks/tuya/mcu/__init__.py

```diff
@@ -548,6 +563,154 @@ class MoesSwitchManufCluster(TuyaOnOffManufCluster):
     data_point_handlers.update({15: "_dp_2_attr_update"})
 
 
+class TuyaNewManufClusterForWindowCover(TuyaMCUCluster):
+    """Manufacturer Specific Cluster for cover device (based on new TuyaMCUCluster).
+
+    I.e. Uses newer TuyaMCUCluster mechanism for translations between cluster attributes and
+    Tuya data points.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
```

Comment:
This can be removed.

Reply:
Done. Although it's a bit awkward if the DPToAttributeMapping converter needs to refer to self. I worked around it by instead using data_point_handlers, which finds a function by name and will call an instance method.